### PR TITLE
MB-9256 update service items calculations UI

### PR DESCRIPTION
**Pull request by YanZ777**
## Description

Bold the weights accordingly in the service item parameters.

## Reviewer Notes

I'm not sure what the shipment weights correspond to? Should that be taken out completely? 
Is the bolding happening at the correct times?
It might take a while for the payment requests to load.
I had to refactor quite a bit to get the bolded styling on the details, so that's why there's so many changes. 
Added some devseed data to the PARAMS move so that's why there's so many things to approve now. 

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make office_client_run
make server_run
make db_dev_e2e_populate
```

Use the move with the locator PARAMS.
The fastest way might be to approve all of them and then look on the payment requests page for what they look like. 
You can also expand the calculations for all the ones that are domestic linehaul.
There's a lot of payment requests to go through.

## Code Review Verification Steps
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-9256) for this change

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

image

image

image

image

image

image

### CODE REVIEW

**Review by kylehilltruss — COMMENTED** [review]
So, I can't speak to any of the functionality changes (which is why I'm not approving this PR), but @YanZ777 and I had talked through a few options of how to practically add conditional styling to the calculation details, and this looks good to me from an implementation perspective.

File: src/components/Office/ServiceItemCalculations/helpers.js [hunk 1]
```diff
@@ -61,20 +61,81 @@ const getPriceRateOrFactor = (params) => {
...
 
-  const weightEstimated = getParamValue(SERVICE_ITEM_PARAM_KEYS.WeightEstimated, params);
-  const weightEstimatedDetail = `${SERVICE_ITEM_CALCULATION_LABELS[SERVICE_ITEM_PARAM_KEYS.WeightEstimated]}: ${
-    weightEstimated ? formatWeight(parseInt(getParamValue(SERVICE_ITEM_PARAM_KEYS.WeightEstimated, params), 10)) : ''
-  }`;
-  return calculation(value, label, weightBilledDetail, weightEstimatedDetail);
+  const weightBilledDetail = formatWeightDetailText(params, SERVICE_ITEM_PARAM_KEYS.WeightBilled);
+  if (weightBilledDetail) {
+    details.push(formatDetail(weightBilledDetail));
+  }
```
Comment (donaldthai) [inline src/components/Office/ServiceItemCalculations/helpers.js:93]: I don't think we need this right? `WeightBilled` is being mapped to `Shipment weight` which is not shown in the new design.

**Review by donaldthai — APPROVED** [review]
Noticed one thing but thanks for working on this! Very clever to pass styles back in the return object!

File: src/components/Office/ServiceItemCalculations/helpers.js (hunk 1 above)
Comment (YanZ777) [reply src/components/Office/ServiceItemCalculations/helpers.js]: I thought so, but I wanted to check. I took it out!

**Review by cosburn — APPROVED** [review]
Looks good!

**Pull request merged**